src/memory.rs

```diff
@@ -34,9 +32,18 @@ impl MemoryUnit {
     }
 }
 
-pub struct Memory {
+#[derive(Serialize, Deserialize)]
+pub struct MemoryConfig {
     header: String,
+    /// Unit used for memory usage. Possible values include KiB, MiB and GiB
+    unit: MemoryUnit,
+    /// Whether to show memory usage as percentage
+    percentage: bool,
+}
+
+pub struct Memory {
     usage: String,
+    config: MemoryConfig,
```

Comment:
is it possible to skip the extra struct here and use `#[serde(skip)]` for the fields we don't want to (de)serialize?